### ISSUE TITLE
Solver config difference between FPN and C4

### ISSUE THREAD

**Issue by yaoliUoA** [facebookresearch/maskrcnn-benchmark#115]
## ❓ Questions and Help
Hi 

I notice that the solver configuration files of two versions for mask-rcnn based on resnet50 (C4 and FPN) are quite different. 

For the C4 model (e2e_mask_rcnn_R_50_C4_1x.yaml):
`SOLVER:
  BASE_LR: 0.01
  WEIGHT_DECAY: 0.0001
  STEPS: (120000, 160000)
  MAX_ITER: 180000
  IMS_PER_BATCH: 8`    

For the FPN model (e2e_mask_rcnn_R_50_FPN_1x.yaml): 
`SOLVER:
  BASE_LR: 0.02
  WEIGHT_DECAY: 0.0001
  STEPS: (60000, 80000)
  MAX_ITER: 90000`

Why the base learning rate for FPN is twice of the C4 version? And why the MAX_ITER is so different too? Are these the recommanded settings for these two versions respectively?

**Comment by fmassa** [comment]
Yes, those values are expected and follow Detectron config files. 

The reason why the values is different is because C4 uses a batch size of 1 per you, while FPN uses 2 imgs per you. This means that you need to apply the lr scaling, as well as the number of iterations, as discussed in the README in the SINGLE GPU training section.

I'm closing this issue, but let me know if you have further questions
